python/pyarrow/tests/util.py

```diff
@@ -110,27 +109,16 @@ def randdecimal(precision, scale):
 
 
 def random_ascii(length):
+    import numpy as np
     return bytes(np.random.randint(65, 123, size=length, dtype='i1'))
```

Comment:
Perhaps this can be fixed to not require NumPy?
```suggestion
    return bytes([random.randint(65, 122) for i in range(length)])
```